corehq/apps/reports/filters/fixtures.py

```diff
@@ -138,5 +138,10 @@ def filter_context(self):
 
         return context
 
+    @classmethod
+    def get_value(cls, request, domain):
```

Comment:
@czue 

I overridden this method from ```BaseReportFilter```, in main class this method working on the filter's ```slug``` and when we use original ```get_value``` on ```AsyncLocationFilter``` then we get ```None``` because in ```AsyncLocationFilter``` ```slug``` is equals to ``` location_async``` but property in request ```GET``` is equals to ```location_id```